lms/templates/calculator/toggle_calculator.html

```diff
@@ -0,0 +1,133 @@
+<%! from django.utils.translation import ugettext as _ %>
+<%! from django.core.urlresolvers import reverse %>
+
+<div class="calc-main">
+    <a title="${_('Open Calculator')}" href="#" role="button" aria-controls="calculator_wrapper" aria-expanded="false" class="calc">${_("Open Calculator")}</a>
```

Comment:
I think that's best. Note that it's not necessarily incorrect to use `<a href="#" role="button">` but using `<button>` simplifies the process. Also, buttons can receive the same styling as links which is nice.

I've seen this a few times across LMS and Studio, so we should harmonize all instances at some point. Maybe we should come to a collective agreement on how to handle these things? I've been of the school "buttons do something, links go somewhere", but we should discuss and all agree. FWIW, I'm going with whatever school is best for accessibility.